framework/aster-frame/src/bus/pci/cfg_space.rs

```diff
@@ -12,74 +14,143 @@ use crate::{
+    /// Capabilities pointer
     CapabilitiesPointer = 0x34,
+    /// Interrupt Line
     InterruptLine = 0x3C,
+    /// INterrupt PIN
     InterruptPin = 0x3D,
+    /// Min Grant
     MinGrant = 0x3E,
+    /// Max latency
     MaxLatency = 0x3F,
 }
 
 bitflags! {
     /// PCI device common config space command register.
     pub struct Command: u16 {
+        /// Sets to 1 if the device can respond to I/O Space accesses.
         const IO_SPACE                  =  1 << 0;
+        /// Sets to 1 if the device can respond to Memory SPace accesses.
         const MEMORY_SPACE              =  1 << 1;
+        /// Sets to 1 if the device can behave as a bus master.
         const BUS_MASTER                =  1 << 2;
+        /// Sets to 1 if the device can monitor Special Cycle operations.
         const SPECIAL_CYCLES            =  1 << 3;
+        /// Memory Write and Invalidate Enable. Set to 1 if the device can
+        /// generate the Memory Write and Invalidate command.
         const MWI_ENABLE                =  1 << 4;
+        /// Sets to 1 if the device does not respond to palette register writes
+        /// and will snoop the data.
         const VGA_PALETTE_SNOOP         =  1 << 5;
+        /// Sets to 1 if the device will takes its normal action when a parity
+        /// error is detected.
         const PARITY_ERROR_RESPONSE     =  1 << 6;
-        const STEPPING_CONTROL          =  1 << 7;
+        /// Sets to 1 if the SERR# driver is enabled.
         const SERR_ENABLE               =  1 << 8;
+        /// Sets to 1 if the device is allowed to generate fast back-to-back
+        /// transactions
         const FAST_BACK_TO_BACK_ENABLE  =  1 << 9;
+        /// Sets to 1 if the assertion of the devices INTx# signal is disabled.
         const INTERRUPT_DISABLE         =  1 << 10;
     }
 }
 
 bitflags! {
     /// PCI device common config space status register.
     pub struct Status: u16 {
+        /// The status of the device's INTx# signal.
         const INTERRUPT_STATUS          = 1 << 3;
+        /// Sets to 1 if the device support capabilities.
         const CAPABILITIES_LIST         = 1 << 4;
+        /// Sets to 1 if the device is capable of running at 66 MHz.
         const MHZ66_CAPABLE             = 1 << 5;
+        /// Sets to 1 if the device can accpet fast back-to-back transactions
+        /// that are not from the same agent.
         const FAST_BACK_TO_BACK_CAPABLE = 1 << 7;
+        /// This bit is only set when the following conditions are met:
+        /// 1. The bus agent asserted PERR# on a read or observed an assertion
+        /// of PERR# on a write
+        /// 2. The agent setting the bit acted as the bus master for the
+        /// operation in which the error occurred
+        /// 3. Bit 6 of the Command register (Parity Error Response bit) is set
+        ///  to 1.
         const MASTER_DATA_PARITY_ERROR  = 1 << 8;
+        /// Read only bits that represent the slowest time that a device will
```

Comment:
Read -> Reads. The starting verb in comments of this bitflags should be in a unified form, all with "s" or all without "s".